views/beneficial-owner-other.html

```diff
@@ -99,110 +124,202 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the corporate benefi
-            text: "County"
+            text: "City or town"
           },
           classes: "govuk-!-width-two-thirds",
-          id: "serviceAddressCounty",
-          name: "serviceAddressCounty",
-          value: serviceAddress.serviceAddressCounty if serviceAddress
+          id: "service_address_town",
+          name: "service_address_town",
+          value: service_address.town if service_address,
+          autocomplete: "service_address_town"
         }) }}
 
         {{ govukInput({
           label: {
-            text: "Postcode"
+            text: "County, state, province or region (optional)"
+          },
+          classes: "govuk-!-width-two-thirds",
+          id: "service_address_county",
+          name: "service_address_county",
+          value: service_address.county if service_address
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Country"
+          },
+          classes: "govuk-!-width-two-thirds",
+          id: "service_address_country",
+          name: "service_address_country",
+          value: service_address.country if service_address
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Postcode or ZIP code (if applicable)"
           },
           classes: "govuk-input--width-10",
-          id: "serviceAddressPostcode",
-          name: "serviceAddressPostcode",
-          value: serviceAddress.serviceAddressPostcode if serviceAddress,
-          autocomplete: "serviceAddressPostcode"
+          id: "service_address_postcode",
+          name: "service_address_postcode",
+          value: service_address.postcode if service_address,
+          autocomplete: "service_address_postcode"
         }) }}
 
         {% endcall %}
         {% endset -%}
 
         {{ govukRadios({
           classes: "govuk-radios",
-          idPrefix: "isSameAddress",
-          name: "isSameAddress",
+          idPrefix: "is_service_address_same_as_principal_address",
+          name: "is_service_address_same_as_principal_address",
           fieldset: {
             legend: {
-              text: "Is their service address the same as their principal office address?",
+              text: "Is its correspondence address the same as its principal or registered office address?",
               isPageHeading: false,
               classes: "govuk-fieldset__legend--m"
             }
           },
-          hint: {
-            text: "A service address is a correspondence address."
-          },
           items: [
             {
-              value: "yes",
+              value: 1,
               text: "Yes",
-              checked: isSameAddress === "yes"
+              checked: is_service_address_same_as_principal_address === "yes"
             },
             {
-              value: "no",
+              value: 0,
               text: "No",
-              checked: isSameAddress === "no",
+              checked: is_service_address_same_as_principal_address === "no",
```

Comment:
Should be `is_service_address_same_as_principal_address == 0` and in the controller we need to cast it to be an integer because `nunjucks` sent just string .
Check [here](https://github.com/companieshouse/overseas-entities-web/blob/34511ad5361850b826d05003646a7b939d1a75ef/src/controllers/beneficial.owner.individual.controller.ts#L33) for an example